bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs/diff/PreliminaryMerge.scala

```diff
@@ -156,6 +158,110 @@ object PreliminaryMerge {
         }
       }
 
-    go(Both(mc.local.prog, mc.remote.plan.update))
+    def addDataObjectConflicts(in: Tree[MergeNode]): TryVcs[Tree[MergeNode]] = {
+      val commonKeys = mc.remote.diffMap.keySet & mc.local.nodeMap.keySet
+
+      def dobConflicts(k: SPNodeKey): Boolean = {
+        lazy val local  = mc.local.nodeMap(k).getDataObject
```

Comment:
What's the benefit of being lazy here? I always think that `lazy` is a rare situation to be in